notebooks/Benchmark_notebook.ipynb

```diff
@@ -54,20 +54,33 @@
    "outputs": [],
    "source": [
     "e = 2.7182818284590452353602874 #hardcoded to avoid dependence on torch / numpy\n",
+    "runtime_f = 0 #Runtime global variable\n",
```

Comment:
I think you need to set this 0 again in the `evaluate` function? Otherwise it will never be reset, if I am not mistaken